.github/workflows/go-integration.yml

```diff
@@ -20,18 +19,46 @@ jobs:
         # both versions in the matrix -- this will lead to them overwriting
         # each other.
         go: [ '1.18' ]
+    # As of December 2022, using 8vcpu runners is slower overall,
+    # due to longer queue times.
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    env:
+      ARMADA_EXECUTOR_INGRESS_URL: "http://localhost"
+      ARMADA_EXECUTOR_INGRESS_PORT: 5001
+      # Cache Docker layers in the Github actions cache.
+      # These variables are picked up by the goreleaser config.
+      DOCKER_BUILDX_CACHE_FROM: "type=gha"
+      DOCKER_BUILDX_CACHE_TO: "type=gha,mode=max"
+      DOCKER_BUILDX_BUILDER: "builder"
     steps:
-      - uses: actions/checkout@v3.1.0
-      - uses: ./.github/workflows/go-setup
-      - run: make build-ci
-      - run: make tests-e2e-setup
-      - run: make tests-e2e-no-setup
-        env:
-          INTEGRATION_ENABLED: true
-      - run: make junit-report
-      - name: Upload junit report
+      - uses: actions/checkout@v3
+      # Needed for docker/setup-buildx-action; see
+      # https://github.com/moby/buildkit#github-actions-cache-experimental
+      - name: Expose GitHub Runtime
+        uses: crazy-max/ghaction-github-runtime@v2
```

Comment:
guess we've done our due diligence on this image!

Reply:
Let me try removing this to see if it still works.